todo_app/app.py

```diff
@@ -8,7 +11,46 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+   items = get_items()
+   sorted_items = sorted(items, key = itemgetter('status'), reverse=True)
+   return render_template('index.html', title='To Do App', items=sorted_items)
+
+
+@app.route('/items/add', methods = ['POST', 'GET'])
```

Comment:
Your site only makes POST requests to this path. You can remove the 'GET' from the list of methods, and remove the if request.method == 'POST' check.

Same for your items/complete route.